### ISSUE TITLE
Publish arm images

### ISSUE THREAD

**Issue by mightyguava** [EnricoMi/publish-unit-test-result-action#510]
I'm unable to use this action on self-hosted arm runners, due to the following error

```
/usr/bin/docker pull ghcr.io/enricomi/publish-unit-test-result-action:v2.10.0
  v2.10.0: Pulling from enricomi/publish-unit-test-result-action
  no matching manifest for linux/arm6[4](https://github.com/squareup/wallet/actions/runs/6424987352/job/17448516682#step:2:4)/v8 in the manifest list entries
```

Would you be able to push arm based images as well? I think all that needs to be done is to change the `build-push` action [here](https://github.com/EnricoMi/publish-unit-test-result-action/blob/master/.github/workflows/ci-cd.yml#L139C15-L139C42) to

```
uses: docker/build-push-action@v5
with:
  platforms: linux/amd64,linux/arm64
```

**Comment by EnricoMi** [comment]
I would prefer publishing images that I can test, but I have no capabilities for testing arm. Can you suggest some way to test those images?

**Comment by mightyguava** [comment]
Sure, https://github.com/multiarch/qemu-user-static can be used to enable qemu emulation for arm on x86. It should allow you to run the arm version of the image on an x86 machine.

If you want to do so from within github actions, running this action before the arm action you want to run should work

```
    - name: Set up QEMU
      uses: docker/setup-qemu-action@v3
      with:
        image: tonistiigi/binfmt:latest
        platforms: arm64
```

**Comment by EnricoMi** [comment]
Thanks for the pointers! Fixed in #512, tested in #513.

**Comment by EnricoMi** [comment]
This has been released.
